sdk/monitor/Azure.Monitor.Ingestion/TROUBLESHOOTING.md

```diff
@@ -58,7 +58,7 @@ When you send logs to Azure Monitor for ingestion, the request may succeed, but
 
 * Verify that the custom table specified in the DCR exists in the Log Analytics workspace. Ensure that you provide the accurate name of the custom table to the upload method. Mismatched table names can lead to logs not being stored correctly.
 
-* Confirm that the logs you're sending adhere to the format expected by the DCR. The data should be in the form of a JSON object or array, structured according to the requirements specified in the DCR. Additionally, it's essential to encode the request body in UTF-8 to avoid any data transmission issues.
+* Confirm that the logs you're sending adhere to the format expected by the DCR. The data should be an array of JSON objects, structured according to the requirements specified in the DCR. Additionally, it's essential to encode the request body in UTF-8 to avoid any data transmission issues.
```

Comment:
There is also an option to specify the serializer in the options bag so the user is not tied to Json
    public ObjectSerializer Serializer { get; set; }